src/validateRegisterForm.js

```diff
@@ -8,32 +8,39 @@
  * @returns {object}
  */
 function validateRegisterForm(email, password) {
-  // eslint-disable-next-line max-len
-  const validPassword = /^(?=.*\d)(?=.*[a-z])(?=.*[A-Z])(?=.*[^a-zA-Z0-9])(?!.*\s).{8,16}$/;
+  // eslint-disable-next-line
+  const validPassword =
+    /^(?=.*\d)(?=.*[a-z])(?=.*[A-Z])(?=.*[^a-zA-Z0-9])(?!.*\s).{8,16}$/;
 
-  // eslint-disable-next-line max-len
-  const validEmailMask = new RegExp(/^([\w-]+(?:\.[\w-]+)*)@((?:[\w-]+\.)*\w[\w-]{0,66})\./i);
+  // eslint-disable-next-line
+  const validEmailMask = new RegExp(
+    /^([\w-]+(?:\.[\w-]+)*)@((?:[\w-]+\.)*\w[\w-]{0,66})\./i
+  );
 
   if (!email.match(validEmailMask) && password.match(validPassword)) {
```

Comment:
The condition `!email.match(validEmailMask) && password.match(validPassword)` seems incorrect. It should be `!email.match(validEmailMask) && !password.match(validPassword)` to correctly identify when both email and password are invalid.